foundation/foundation-upgrade/src/test/java/com/boozallen/aissemble/upgrade/migration/version_specific/MlTrainImageTagMigrationSteps.java

```diff
@@ -0,0 +1,57 @@
+package com.boozallen.aissemble.upgrade.migration.version_specific;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractMigrationTest;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import java.io.File;
+import java.nio.file.Path;
+
+public class MlTrainImageTagMigrationSteps extends AbstractMigrationTest {
+    public static final String MIGRATION_ROOT = "version-specific/MlTrainImageTagMigration/migration/ml-training/%svalues-dev.yaml";
+
+    @Given("a ML training pipeline values-dev file that does not have the imagePullPolicy and tag set")
+    public void a_ml_training_pipeline_values_dev_file_that_does_not_have_the_image_pull_policy_and_tag_set(){
+        testFile = getTestFile(String.format(MIGRATION_ROOT, ""));
```

Comment:
S: use abstract method
```suggestion
        setTestFileToVersionMigration("MlTrainImageTagMigration", "values-dev.yaml");
```

Reply:
removed, this was part of the unused step